Create FAQ.cs

ssbtoday opened the pull request.

ssbtoday committed:
Create FAQ.cs

ssbtoday committed:
Merge branch 'master' of https://github.com/Rithari/OnePlusBot
Move FAQ.cs to new folder.

Rithari commented:
Adding blusparks exclusively does not make this a FAQ, try adding more questions, also mistyping should list all available options.

ssbtoday committed:
Updated FAQ.cs

Added additional FAQ modules, OxygenOS Mirrors, Google Camera, Unbrick Guides, and added additional alias to bluspark.

ssbtoday commented:
@Rithari Updated per your request.
